parser/prism/Translator.cc

```diff
@@ -1036,15 +1079,25 @@ std::unique_ptr<parser::Node> Translator::translateConstantPath(pm_constant_path
         parent = translate(node->parent);
     } else {                                                // This is a fully qualified constant reference, like `::A`.
         pm_location_t *delimiterLoc = &node->delimiter_loc; // The location of the `::`
-        parent = make_unique<parser::Cbase>(parser.translateLocation(delimiterLoc));
+        parent = make_node<parser::Cbase>(parser.translateLocation(delimiterLoc));
```

Comment:
In retrospect, `parent` is a kind of ambiguous term here. To clarify, it's a parent part of the path, e.g. the `A` in `A::B`, but it's a child node, as far as the Prism AST structure goes.